### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent error on multi-line function call with static closure argument

### ISSUE THREAD

**Issue by hews** [squizlabs/PHP_CodeSniffer#1761]
When passing an anonymous function to another function (eg, `array_map()`), and putting both the called arguments on new lines, `Generic.WhiteSpace.ScopeIndent` identifies the arguments and correctly allows the following:

```php
array_map(
    function ( $item ) {
        echo $item;
    },
    $some_array
);
```

However, when using the `static` keyword on the anonymous function, as in:

```php
 1 | array_map(
 2 |     static function ( $item ) {
 3 |         echo $item;
 4 |     },
 5 |     $some_array
 6 | );
```

The following error is triggered:

```
  2 | ERROR | [x] Line indented incorrectly; expected 0 tabs, found 1 (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
```

**Comment by Jasu** [comment]
Came here to report the same bug - this occurs with PHPCS 3.1.1.

**Comment by gsherwood** [comment]
Error was caused by the sniff enforcing exact indent rules for method prefixes, but didn't ignore this for closures. Now it does, and the error is resolved.

Thanks for reporting this.
